js/src/jsonrpc/interfaces/parity.js

```diff
@@ -1622,7 +1650,14 @@ export default {
         example: {
           from: '0xb60e8dd61c5d32be8058bb8eb970870f07233155',
           to: '0xd46e8dd67c5d32be8058bb8eb970870f07244567',
-          value: fromDecimal(2441406250)
+          gas: fromDecimal(30400),
```

Comment:
Would just remove the `fromDecimal` wrappers here, i.e. the response should rather be what we receive via the actual RPC, raw.